GameServerScripts/dbupdater/GuildAndAllianceUpdate.cs

```diff
@@ -42,12 +42,12 @@ public void Update()
 				log.Info("Start Searching for records that need update...");
 			
 			// Change the Leader Relation if Missing
-			var alliances = DOLDB<DBAlliance>.SelectObjects(DB.Column("LeaderGuildID").IsEqualTo(string.Empty).Or(DB.Column("LeaderGuildID").IsNull()));
+			var alliances = DOLDB<DBAlliance>.SelectObjects(DB.Column(nameof(DBAlliance.LeaderGuildID)).IsEqualTo(string.Empty).Or(DB.Column(nameof(DBAlliance.LeaderGuildID)).IsNull()));
```

Comment:
The usage of `nameof` in GameServerScripts does not work because it is using an older compiler unfortunately. The same applies for L50 in the same file of course.

Reply:
Thanks, I have updated the PR and excluded GameServerScripts from the use of nameof.
Additionally I have created another commit to change the names of the relation columns in the DB.
Still I miss the _ID column name but that would require to have it computed at compile time.